gcb/presubmit.sh

```diff
@@ -62,6 +62,16 @@ bazel_run() {
 # Verify that all Bazel files are formatted correctly.
 $BUILDIFIER -mode=check -r `pwd`
 
+# Verify that all Bazel build files have a license rule. Exclude files in the
+# gcb directory.
+# BUILD_FILES_WITHOUT_LICENSES=$(fgrep -L 'licenses(' $(find . -name BUILD) | grep -v gcb)
+# NUM_BUILD_FILES_WITHOUT_LICENSES=$(echo "$BUILD_FILES_WITHOUT_LICENSES" | wc -w)
+# if [ "$NUM_BUILD_FILES_WITHOUT_LICENSES" -ne 0 ]; then
+#   echo "Found build files without licenses:"
+#   echo "$BUILD_FILES_WITHOUT_LICENSES"
+#   return 1
+# fi
```

Comment:
stale?

Reply:
No, I'm trying to figure out why the build was failing. I committed the presubmit code commented out to check to see whether it was the cause of the problem. Still waiting on the results there.

Reply:
Oh, you want to make this part of the presubmit check? Sounds good.

Reply:
Thank you for your help debugging this, this is in and working (on my Docker image anyway).